Makefile

```diff
@@ -34,7 +34,7 @@ ebml_matroska_elements4rfc.md: transforms/ebml_schema2markdown4rfc.xsl matroska_
 	xsltproc transforms/ebml_schema2markdown4rfc.xsl matroska_xsd.xml > $@
 
 $(OUTPUT_MATROSKA).md: index_matroska.md diagram.md matroska_schema_section_header.md ebml_matroska_elements4rfc.md ordering.md chapters.md attachments.md cues.md streaming.md menu.md notes.md rfc_backmatter_matroska.md
-	cat $^ | grep -v '^---' > $@
+	sed -e '/^---/,/^---/d' $^ > $@
```

Comment:
This is probably better but it's probably still not goo enough. Any text in between the two `---` at the top the file will be kept. Luckily for now we don't use this section at all.